src/api/app/views/layouts/webui/responsive_ux/_sign_up.html.haml

```diff
@@ -0,0 +1,19 @@
+.signup-form.d-none
+  .h2.text-center.py-2 Create an account
+  = form_tag(users_path, method: :post, class: 'sign-up', autocomplete: 'off') do
```

Comment:
What about `proxy_mode`?

Reply:
`proxy_mode` doesn't need a Sign Up form. It only has a link to a page defined in the configuration. Does it answer your question?